winpr/libwinpr/crt/unicode.c

```diff
@@ -169,7 +168,7 @@ int MultiByteToWideChar(UINT CodePage, DWORD dwFlags, LPCSTR lpMultiByteStr,
 		sourceStart = (const BYTE*) lpMultiByteStr;
 		targetStart = (WCHAR*) NULL;
 
-		result = ConvertUTF8toUTF16(&sourceStart, &sourceStart[cbMultiByte],
+		ConvertUTF8toUTF16(&sourceStart, &sourceStart[cbMultiByte],
```

Comment:
Again, the problem isn't that the result variable exists in the first place, the problem is that it is unchecked! To get rid of the warnings, check variables, don't remove them please.